package.json

```diff
@@ -57,6 +57,7 @@
     "@types/minimist": "1.2.1",
     "@types/mocha": "8.2.1",
     "@types/node": "14.14.32",
+    "0x": "4.10.2",
```

Comment:
This was inadvertently added by another PR: https://github.com/ampproject/amphtml/pull/33111/files#r591649903. Can we remove it here?

Reply:
Oh, I thought that change was intentional. Removed.

Reply:
This is now being cleaned up in #33306.